src/translator/byte_array_util.cpp

```diff
@@ -27,8 +25,8 @@ const T* get(const void*& current, uint64_t num = 1) {
   return ptr;
 }
 
-bool validateBinaryModel(AlignedMemory& model, uint64_t fileSize) {
-  const void * current = &model[0];
+bool validateBinaryModel(const AlignedMemory *model, uint64_t fileSize) {
+  const void * current = model->begin();
```

Comment:
What's happening here? Aren't references preferable over pointers because the former cannot be null?

Reply:
A void reference seems icky to me.

Reply:
I was trying to call `validateBinaryModel` in `BatchTranslator` where the model bytes is passed as const AlignedMemory*. I will change to `const AlignedMemory &model`.